### PR TITLE
Move sed code before LSMB start

### PR DESCRIPTION
**Pull request by ylavoie**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -116,6 +116,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
+  - if [ "${DOJO_BUILT}" != "" ]; then
+      sed -r -i -e "s/^( |\t|#)*dojo_built( |\t)*=.+/dojo_built = ${DOJO_BUILT}/" ledgersmb.conf ;
```
Comment (sbts) [inline .travis.yml]: )*=.+/
needs to be
)*=.*/

File: utils/test/Is_LSMB_running.sh [hunk 2]
```diff
@@ -82,6 +82,10 @@ if $UPDATE; then
     cp $current $src
 fi
 
+# so we don't fail simply for dojo_built=0 (js-src) causing changes
+# modify the current file to appear to be only js instead of js-src
+sed -i.bak 's|"js-src/|"js/|' "$current"
```
Comment (sbts) [inline utils/test/Is_LSMB_running.sh]: This change isn't needed as it's already in place for master and 1.5

File: .travis.yml [hunk 3]
```diff
@@ -122,6 +122,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
+  - if [ "${DOJO_BUILT}" != "" ]; then
+      sed -r -i -e "s/^( |\t|#)*dojo_built( |\t)*=.*/dojo_built = ${DOJO_BUILT}/" ledgersmb.conf ;
+    fi
```
Comment (sbts) [inline .travis.yml]: Why do we need to do this?
we have 2 ledgersmb.conf files to cover dojo built and unbuilt.
Also we can specify which file is used in an env-var

File: .travis.yml [hunk 4]
```diff
@@ -122,6 +122,9 @@ before_script:
   - tar xzf tar-output/ledgersmb-travis.tar.gz
   - cd ledgersmb
   - cp conf/ledgersmb.conf.default ledgersmb.conf
```
Comment (sbts) [inline .travis.yml]: Rather than copying the config file, we should simply set the LSMB_CONFIG_FILE environment variable

File: .travis.yml (hunk 3 above)
Comment (ylavoie) [inline .travis.yml]: They are out of sync. Maintaining them will always be a problem. I advocate using only one file and toggling the option.

Comment (sbts) [inline .travis.yml]: I'll solve the maintenance issue with a test to confirm all 3 are in sync (default, unbuilt, and travis)
at the same time, I'll make sure they are all in synv

**Comment by ylavoie** [comment]
Already handled.

**Pull request closed without merging**